src/framework/ui/dropdown/dropdown.component.tsx

```diff
@@ -51,22 +51,22 @@ type ControlElement = React.ReactElement<TouchableOpacityProps>;
 type IconProp = (style: ImageStyle, visible: boolean) => IconElement;
 type DropdownChildren = [DropdownMenuElement, TextElement, ControlElement];
 
-export type SelectedOption = Array<DropdownItemType> | DropdownItemType;
+export type DropdownOption = Array<DropdownItemType> | DropdownItemType;
 
 const MEASURED_CONTROL_TAG: string = 'Control';
 
 interface ComponentProps {
   data: DropdownItemType[];
   multiSelect?: boolean;
-  selectedOption?: SelectedOption;
+  selectedOption?: DropdownOption;
   textStyle?: TextStyle;
   placeholder?: string;
   placeholderStyle?: StyleProp<TextStyle>;
   label?: string;
   labelStyle?: StyleProp<TextStyle>;
   controlStyle?: StyleProp<ViewStyle>;
   icon?: IconProp;
-  onSelect: (option: SelectedOption, event?: GestureResponderEvent) => void;
+  onSelect: (option: DropdownOption, event?: GestureResponderEvent) => void;
   size?: string;
   status?: string;
   appearance?: string;
```

Comment:
has declaration in `StyledComponentProps`

Reply:
Removed